doc/src/Installing.rst

```diff
@@ -54,73 +106,116 @@ The following packages are optional:
 
 * `Readline <https://tiswww.case.edu/php/chet/readline/rltop.html>`_
 * `Ncurses <https://invisible-island.net/ncurses/>`_ (progress bar display)
-* `LevMar <http://users.ics.forth.gr/~lourakis/levmar/>`_ (model fitting)
+* `LevMar <http://users.ics.forth.gr/~lourakis/levmar/>`_ (model fitting engine)
+* `GSL <https://www.gnu.org/software/gsl/>`_ (model fitting engine)
 
-All or most of these packages are available in the main linux distributions.
+All or most of these packages are available in the main Linux distributions.
 You will also need the ``-devel`` version of the packages if you compile |SourceXtractor++| from the source.
 
+If you have installed Elements and Alexandria from your distribution repository
+(right now, only in Fedora), you can skip the following section.
+
+.. code-block:: console
+
+  $ dnf install elements-devel elements-alexandria-devel
+
+
+Installation of Elements and Alexandria
+_______________________________________
+
+Elements is a C++ and CMake framework. It is capable of managing the dependencies
+of the projects that are based on it, as long as it knows where to find them.
+By convention, this location usually is ``~/Work/Projects``, but you can choose
+any other.
```

Comment:
What convention is that? Isn't `/usr` the usual prefix (at least for system-wide installation)?

Reply:
Element's convention, if you want to build everything without installing into the system. This works without being root, for instance.